libs/entry-components/styles/partials/vendors/overrides/rich-text/_forms.scss

```diff
@@ -0,0 +1,20 @@
+@use '@angular/material' as mat;
+
+$checkbox-width: 18px;
```

Comment:
Since the Redactor component generates its own DOM elements and doesn’t directly use Angular Material components, it can’t directly inherit Angular Material's checkbox dimensions. Additionally, Angular Material doesn’t expose the checkbox width and height as SCSS variables, so to avoid any confusion, the most practical solution is to set custom SCSS variables to override redactor checkbox width and height that approximate Angular Material’s dimensions.
 ![image](https://github.com/user-attachments/assets/09766651-9554-4e5f-8cc2-b8e3cb70c6ee)